app/bundles/PageBundle/EventListener/PageSubscriber.php

```diff
@@ -188,13 +188,20 @@ public function onPageHit(QueueConsumerEvent $event)
         $trackingNewlyGenerated = $payload['isNew'];
         $pageId                 = $payload['pageId'];
         $leadId                 = $payload['leadId'];
+        $isRedirect             = $payload['isRedirect'];
```

Comment:
Let's put a `!empty()` around this to prevent any issues for jobs that were injected right before an update where isRedirect will not be defined.